src/main/java/dk/kb/present/copyright/CopyrightAccessExtractor.java

```diff
@@ -0,0 +1,364 @@
+package dk.kb.present.copyright;
+
+import java.io.StringReader;
+import java.util.ArrayList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+
+import dk.kb.present.copyright.CopyrightAccessDto.AccessCondition;
+
+import dk.kb.present.copyright.CopyrightAccessDto.CreatorCorporate;
+import dk.kb.present.copyright.CopyrightAccessDto.CreatorPerson;
+import dk.kb.present.webservice.exception.InvalidArgumentServiceException;
+import dk.kb.util.xml.XMLEscapeSanitiser;
+
+/*
+ * This document is the bible for what is going on:
+ * https://kbintern.sharepoint.com/:w:/r/sites/Proj-KULA-186-Digitale-samlinger-Amanda-Britta/_layouts/15/Doc.aspx?action=edit&sourcedoc=%7B915b7ba6-eeae-4636-b04c-472b83aa81f6%7D&wdOrigin=TEAMS-ELECTRON.teamsSdk.openFilePreview&wdExp=TEAMS-CONTROL&web=1&cid=5b8502b8-d6cc-4d97-8191-7f3abe6e3c5b
+ * This should be the newest version and is stil not finished. (materiel type logic missing)
+ * The version used to implement this class can be found in the /doc folder in the project
+ * 
```

Comment:
Plus points for the documentation reference. Please also add a line or two about what the class really does (extract copyright and access information from METS/MODS from the Cumulus data @ kb.dk).

Reply:
fixed.

Reply:
Much better.